### PR TITLE
add pbtk

### PR DESCRIPTION
**Pull request by Kincekara**
PacBio BAM toolkit. One of the components related with #926 

Pull Request (PR) checklist:
- [x] Include a description of what is in this pull request in this message.
- [x] The dockerfile successfully builds to a test target for the user creating the PR. (i.e. `docker build --tag samtools:1.15test --target test docker-builds/samtools/1.15` )
- [x] Directory structure as name of the tool in lower case with special characters removed with a subdirectory of the version number (i.e. `spades/3.12.0/Dockerfile`)
   - [ ] (optional) All test files are located in same directory as the Dockerfile (i.e. `shigatyper/2.0.1/test.sh`)
- [x] Create a simple container-specific [README.md](https://github.com/StaPH-B/docker-builds/blob/master/.github/workflow-templates/readme-template.md) in the same directory as the Dockerfile (i.e. `spades/3.12.0/README.md`)
   - [ ] If this README is longer than 30 lines, there is an explanation as to why more detail was needed
- [x] Dockerfile includes the recommended [LABELS](https://github.com/StaPH-B/docker-builds/blob/master/dockerfile-template/Dockerfile#L8-L18) 
- [x] Main [README.md](https://github.com/StaPH-B/docker-builds/blob/master/README.md) has been updated to include the tool and/or version of the dockerfile(s) in this PR
- [x] [Program_Licenses.md](https://github.com/StaPH-B/docker-builds/blob/master/Program_Licenses.md) contains the tool(s) used in this PR and has been updated for any missing

### CODE REVIEW

**Comment by erinyoung** [comment]
It looks like the tests worked
```
#9 [test 1/2] RUN wget -q --no-check-certificate https://downloads.pacbcloud.com/public/dataset/HiFiViral/Nov_2021/m64012_210821_095651.hifi_reads.bam
#9 DONE 380.8s

#10 [test 2/2] RUN pbindex m64012_210821_095651.hifi_reads.bam &&    zmwfilter --include 1,2,4,8,16 m64012_210821_095651.hifi_reads.bam test.bam &&    zmwfilter --include 18,19,22,24,33 m64012_210821_095651.hifi_reads.bam test2.bam &&    pbindex test.bam &&    pbindexdump test.bam.pbi > test.json &&    bam2fasta -o test test.bam &&    bam2fastq -o test test.bam &&    extracthifi test.bam test.hifi.bam &&    ccs-kinetics-bystrandify test.bam test.bystrand.bam &&    pbmerge test.bam test2.bam > test.merged.bam &&    echo $(ls | grep test)
#10 15.53 test.bam test.bam.pbi test.bystrand.bam test.bystrand.bam.pbi test.fasta.gz test.fastq.gz test.hifi.bam test.json test.merged.bam test2.bam
#10 DONE 15.5s
```

File: pbtk/3.1.1/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+ENV LC_ALL=C
+
+CMD bam2fasta -h;\
+    bam2fastq -h;\ 
+    ccs-kinetics-bystrandify -h;\
+    extracthifi -h;\
+    pbindex -h;\
+    pbindexdump -h;\
+    pbmerge -h;\
+    zmwfilter -h
```
Comment (erinyoung) [inline pbtk/3.1.1/Dockerfile:32]: Do these print to stderr?

Comment (Kincekara) [reply pbtk/3.1.1/Dockerfile:32]: Probably, but I am not sure.

**Review by erinyoung — APPROVED** [review]
Thank you for putting this together!

I don't have any changes to recommend.

**Comment by erinyoung** [comment]
You can check the status of the deploy here: https://github.com/StaPH-B/docker-builds/actions/runs/8482703034

**Pull request merged**